catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplication.java

```diff
@@ -181,7 +181,7 @@ public void start() {
               @Override
               public void stop() {
                 long startTime = System.currentTimeMillis();
-                LOG.info("Took " + (System.currentTimeMillis() - startTime) + " ms to close all the services");
+                LOG.info(String.format("Took %s ms to close all the services", (System.currentTimeMillis() - startTime)));
               }
```

Comment:
@MustaphaGHA, you can avoid String.format altogether by using parameterized logging as follows:
LOG.info("Took {} ms to close all the services", (System.currentTimeMillis() - startTime)));

See https://stackoverflow.com/questions/30975026/log4j-implicit-string-formatting

This can be done in other places as well.

Reply:
Alright , applying changes now